vault/migration-bitwarden.sh

```diff
@@ -1,55 +1,64 @@
 #!/usr/bin/env bash
-
+set -e
 
 if [[ -z $VAULT_TOKEN ]]; then
-  echo "Please set the varaible $VAULT_TOKEN"
+  echo "Please set the variable $VAULT_TOKEN"
```

Comment:
```suggestion
  echo "Please set the variable VAULT_TOKEN"
```

Reply:
Why are there so many variables?

- `VAULT_TOKEN`
- `VAULT_HTTP_TOKEN`
- `VAULT_CLIENT_KEY`

Are they all the same thing?

Reply:
`VAULT_CLIENT_KEY` is the private key of the client certificate exchange during call to the API.
`VAULT_TOKEN` is your personal token

Reply:
Ah, thanks for explaining.